code/modules/mob/living/carbon/xenomorph/castes/runner/abilities_runner.dm

```diff
@@ -317,6 +321,25 @@
 
 	return ..()
 
+///Dodging thrown projectiles
+/datum/action/xeno_action/evasion/proc/evasion_throw_dodge(datum/source, atom/movable/proj)
+	SIGNAL_HANDLER
```

Comment:
This autodoc comment needs to explain what the proc does

Also this proc should return NONE not FALSE